### ISSUE TITLE
Broken picture links on Outbound OSS guide

### ISSUE THREAD

**Issue by winterrocks** [todogroup/todogroup.org#353]
It seems that there are several broken (404) picture links in the Outbound OSS md page: 
https://github.com/todogroup/todogroup.org/blob/main/content/en/guides/outbound-oss.md

**Comment by cornelius** [comment]
That's true. The references are done in a way that they are correct in the rendered version at https://todogroup.org/guides/outbound-oss/, but that breaks them in the source view in the repo.

I'm not sure, if there is a good way, which makes the links work in both versions.

**Comment by node** [comment]
If you visit from GitHub repos ,it's true, you will get 404,because the image uri prefix actually is /static/img/guides/ , but when we publish the website, the image uri prefix is /img/guides/ and it works well. 

So I think everything is OK now.
